table/server/common/src/main/java/alluxio/table/common/udb/UdbBypassSpec.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.table.common.udb;
+
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Tables and partitions bypassing specification.
+ */
+public final class UdbBypassSpec {
+  private final Map<String, Set<String>> mTablePartMap;
+
+  /**
+   * @param tablePartMap table to partition map
+   */
+  public UdbBypassSpec(Map<String, Set<String>> tablePartMap) {
+    mTablePartMap = tablePartMap;
+  }
+
+  /**
+   * Checks if a table should be bypassed.
+   *
+   * @param tableName the table name
+   * @return true if the table is configured to be bypassed, false otherwise
+   * @see UdbBypassSpec#isFullyBypassedTable(String)
+   */
+  public boolean isBypassedTable(String tableName) {
```

Comment:
nit: since this class is already called `UdbBypassSpec`, you can simply name this method `hasTable`